src/main/java/com/azure/cosmos/examples/changefeed/SampleChangeFeedProcessor.java

```diff
@@ -0,0 +1,269 @@
+            //The next line causes the worker to create and start an instance of the Change Feed Processor. See the implementation of getChangeFeedProcessor() for guidance
+            //on creating a handler for Change Feed events. In this stream, we also trigger the insertion of 10 documents on a separate
+            //thread.
+            logger.info("-->START Change Feed Processor on worker (handles changes asynchronously)");
+            changeFeedProcessorInstance = getChangeFeedProcessor("SampleHost_1", feedContainer, leaseContainer);
+            changeFeedProcessorInstance.start()
+                    .subscribeOn(Schedulers.elastic())
+                    .doOnSuccess(aVoid -> {
+                        //pass
+                    })
+                    .subscribe();
+
+            //These two lines model an application which is inserting ten documents into the feed container
+            logger.info("-->START application that inserts documents into feed container");
+            createNewDocumentsCustomPOJO(feedContainer, 10, Duration.ofSeconds(3));
+            isWorkCompleted = true;
+
+            //This loop models the Worker main loop, which spins while its Change Feed Processor instance asynchronously
+            //handles incoming Change Feed events from the feed container. Of course in this sample, polling
+            //isWorkCompleted is unnecessary because items are being added to the feed container on the same thread, and you
+            //can see just above isWorkCompleted is set to true.
+            //But conceptually the worker is part of a different thread or application than the one which is inserting
+            //into the feed container; so this code illustrates the worker waiting and listening for changes to the feed container
+            long remainingWork = WAIT_FOR_WORK;
+            while (!isWorkCompleted && remainingWork > 0) {
+                Thread.sleep(100);
+                remainingWork -= 100;
+            }
+
+            //When all documents have been processed, clean up
+            if (isWorkCompleted) {
+                if (changeFeedProcessorInstance != null) {
+                    changeFeedProcessorInstance.stop().subscribe();
+                }
+            } else {
+                throw new RuntimeException("The change feed processor initialization and automatic create document feeding process did not complete in the expected time");
+            }
+
+            logger.info("-->DELETE sample's database: " + DATABASE_NAME);
+            deleteDatabase(cosmosDatabase);
+
+            Thread.sleep(500);
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        logger.info("END Sample");
+    }
+
+    public static ChangeFeedProcessor getChangeFeedProcessor(String hostName, CosmosAsyncContainer feedContainer, CosmosAsyncContainer leaseContainer) {
```

Comment:
For each individual method it will be great if we can capture a short summary of what it is doing.